packages/wrangler/src/deployments.ts

```diff
@@ -65,13 +58,24 @@ export async function deployments(
 		params
 	);
 
-	const versionMessages = deploys.map(
-		(versions) =>
-			`\nDeployment ID: ${versions.id}
-Created on: ${versions.metadata.created_on}
-Author: ${versions.metadata.author_email}
-Source: ${sourceStr(versions.metadata.source)}\n`
-	);
+	const versionMessages = deploys.map((versions) => {
+		const t = versions.annotations?.["workers/triggered_by"]
+			? `${triggerStr(
+					versions.annotations["workers/triggered_by"]
+			  )} from ${sourceStr(versions.metadata.source)}`
+			: `${sourceStr(versions.metadata.source)}`;
+
+		let version = `\nDeployment ID: ${versions.id}
```

Comment:
This could probably benefit from using an implicit newline, as it would make the alignment of values more visible